installtests/service-check.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+# Verify that linux init services correctly handle starting and stopping Jenkins
+set -o
```

Comment:
No, I just just 'set -o nonunset' and didn't remove the whole thing when taking it out.  Honest mistake :-(